Makefile

```diff
@@ -20,12 +20,16 @@ CPPFLAGS += --std=c++14 $(INCLUDES)
 .PHONY: build libcryptopp libff libsecp256k1
 build: client-c/json.o client-c/main.o plugin-c/blake2.o plugin-c/blockchain.o plugin-c/crypto.o plugin-c/plugin_util.o plugin-c/world.o
 
-plugin-c/blockchain.o: plugin/proto/msg.pb.h
+plugin-c/blockchain.o: plugin-c/proto/msg.pb.h
 
 %.pb.h: %.proto
 	protoc --cpp_out=. $<
 
-.PHONY: clean
+.PHONY: install clean
+install:
+	@mkdir -p $(PREFIX)
+	cp plugin/* $(PREFIX)
```

Comment:
`install: ...` should have dependencies on the targets that it actually intends to copy over.

So it should look more like this:

```
INSTALL_PREFIX := /usr/local
INSTALL_DIR   ?= $(DESTDIR)$(INSTALL_PREFIX)/bin

install: $(INSTALL_DIR)/file1 $(INSTALL_DIR)/file2 $(INSTALL_DIR)/file3

$(INSTALL_DIR)/file1 $(INSTALL_DIR)/file2: plugin/file1 plugin/file2
    cp plugin/file1 $(INSTALL_DIR)/file1
    cp plugin/file2 $(INSTALL_DIR)/file2

$(INSTALL_DIR)/file3: other_directory/file3
    cp other_directory/file3 $(INSTALL_DIR)/file3
```

for example.

Reply:
Then the user can modify `INSTALL_PREFIX` to tell it to install somewhere else, and package systems can modify `DESTDIR` to sandbox the builds.

Reply:
On the KEVM repo, do `git show a85adbc3:Makefile` to see how we set this up for KEVM so that packaging for Debian/Ubuntu/Arch was easy using `make install`. It's since been removed from the build system when we stopped packaging KEVM and started packaging Firefly.